index.html

```diff
@@ -292,7 +292,38 @@ <h3>
       <li>Let |providers| be |options|'s {{CredentialRequestOptions/digital}}'s
       {{DigitalCredentialRequestOptions/providers}} member.
       </li>
-      <li>If |providers| is empty, [=exception/throw=] a {{TypeError}}.
+      <li>If |providers| [=list/is empty=], [=exception/throw=] a
+      {{TypeError}}.
+      </li>
+      <li>Let |range| be [=list/get the indices=] of |providers|.
```

Comment:
```suggestion
      <li>Let |range| be [=list/the indices=] of |providers|.
```

Reply:
"get the indices" is an algorithm... but I can alias it.

```suggestion
      <li>Let |range| be [=list/get the indices|the indices=] of |providers|.
```